controllers/fg_handler.go

```diff
@@ -17,17 +17,13 @@ const (
 // Function to handle the feature gates
 func (r *KataConfigOpenShiftReconciler) processFeatureGates() error {
 
-	if r.FeatureGates == nil {
-		return nil
-	}
-
```

Comment:
Hi @bpradipt this seems to be part of the previous commit. Do you mind squashing them?

Reply:
Yeah I'll do it. Are you ok with the logic ?